src/lib/y2storage/proposal/autoinst_devices_creator.rb

```diff
@@ -63,8 +63,10 @@ def populated_devicegraph(planned_devices, disk_names)
         part_creator = Proposal::PartitionCreator.new(original_graph)
         result = part_creator.create_partitions(dist)
 
-        reused.each { |r| r.reuse!(result) }
-        result
+        reused.each { |r| r.reuse!(result.devicegraph) }
```

Comment:
We should maybe rethink a little bit the messages displayed before. I mean those "Partitions to reuse" and "partitions to create".